docs/source/download.rst

```diff
@@ -96,26 +116,38 @@ Windows Subsystem for Linux. There are more detailed
 Installation on previous versions of Windows is unlikely to work.
 
 
-Supported Python distributions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Firedrake requires Python 3.6 or later.
-
-On Ubuntu (18.04 or later), the system installed Python 3 is supported and tested.
-
-On Mac OS, the homebrew_ installed Python 3 is supported and tested::
-
-  brew install python3
-
-If instead you choose to install Python 3 using the official Mac OS
-installer on the Python website, you need to be aware that that
-installation will not have a working SSL by default. You need to
-follow the SSL certificate instructions given in the installation process (or in
-``/Applications/Python X.X/ReadMe.rtf`` after installation).
-
-Additional considerations for MacPorts users
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
+System anti-requirements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We strive to make Firedrake work on as many platforms as we can. Some
+tools, however, make this challenging or impossible for end users.
+
+**Anaconda.** The Anaconda Python distribution and package manager are
+often recommended in introductory data science courses because it does
+effectively handle many aggravating problems of dependency management.
+Unfortunately, Anaconda does a poor job of isolating itself from the
+rest of your system and assumes that it will be both the only Python
+installation and the only supplier of any dependent packages. Anaconda
+will install compilers and MPI compiler wrappers and put its compilers
+right at the top of your `PATH`. This is a problem because Firedrake
```

Comment:
Don't know if you've compiled this but I think ``PATH`` needs two backticks in RST to render as plaintext.